content/v2.0/reference/release-notes/influxdb.md

```diff
@@ -15,6 +15,8 @@ weight: 101
 - Extend `influx stacks update` command with ability to add resources without apply template.
 - Consolidate all InfluxDB template and stack functionality into two new public APIs: `/api/v2/templates` and `/api/v2/stacks`.
 - Extend template `Summary` and `Diff` nested types with `kind` identifiers.
+- Add static builds for Linux.
+- Update Flux to v.0.71.1.
 
```

Comment:
Looks like this is missing the bug fixes that were added: https://github.com/influxdata/influxdb/blob/master/CHANGELOG.md#bug-fixes